sdk/internal/perf/testdata/perf/no_op_perf.go

```diff
@@ -0,0 +1,49 @@
+package main
+
+import (
+	"context"
+
+	"github.com/Azure/azure-sdk-for-go/sdk/internal/perf"
+)
+
+type noOpPerfTest struct {
+	perf.PerfTestOptions
+}
+
+func (n *noOpPerfTest) GlobalSetup(ctx context.Context) error {
```

Comment:
It would be nice if tests could omit all these methods if they don't need to add any functionality.  But I know go doesn't have inheritance like the other languages, so if you think this is the best way to implement this in go, it's fine with me.

Reply:
Unfortunately I'm not sure of a way to do that. Adding a nil method is pretty simple here. When I do a presentation with joel/jeff I'll ask for input.